src/materialized/src/bin/materialized/main.rs

```diff
@@ -668,44 +674,6 @@ max log level: {max_log_level}",
         ),
     }))?;
 
-    eprintln!(
-        "=======================================================================
-Thank you for trying Materialize!
-
-We are interested in any and all feedback you have, which may be able
-to improve both our software and your queries! Please reach out at:
-
-    Web: https://materialize.com
-    GitHub issues: https://github.com/MaterializeInc/materialize/issues
-    Email: support@materialize.com
-    Twitter: @MaterializeInc
-=======================================================================
-"
```

Comment:
I'm going to miss this startup message, but it's indeed not necessary any longer.